cfme/containers/image.py

```diff
@@ -60,17 +61,44 @@ def get_detail(self, *ident):
     def sha256(self):
         return self.id.split('@')[-1]
 
-    def perform_smartstate_analysis(self, wait_for_finish=False, timeout='7M'):
+    @classmethod
```

Comment:
Why is it classmethod?

Reply:
Before my changes , the perform_smartstate_analysis function was call from Image details view when we had single Image instance.
Since my changes allow to work also on multiple images and not just on single image,
Now we use this function from container images view for several Images rows that check from table.
So we don't use specific Image instance for call this function.

Reply:
**REQUIRED**
In that case it should be in `ImageCollection`.

Reply:
You are right about it.
We didn't start our work on collection , and there is no ImageCollection for now.
This will be done after we will finish our web_ui to widgetastic covert.
I add a TODO comment to move this function to ImageCollection.
For now i want to merge this function as is, If its acceptable from your side.

Reply:
@psav, @jkrocil, @mfalesni  what do you say about it?